### DISCUSSION TITLE
Control Playlist / Streams with GPIO´s?

### DISCUSSION THREAD

**Discussion by ghost** [MiczFlor/RPi-Jukebox-RFID#1261] [Q&A] [answered]
Hello,

i have installed a new (and my first) phoniebox with RFID Reader and GPIO´s for Volume and Playercontrol ans everything works fine :-) . My main usage of the box will be "only" a cool Radio that play a few of my favourite Internetstreams or Playlist. Is there a way to control a Playlist/Stream from an GPIO Button instead useing the RFID Reder oder the Webinterface? Like a function in the GPIO Settings like "start ID xxxxxx" and the it does the same like the RFID Card that was learned to the Phoniebox?

Hope my Idea is understandable :-)

Thanks in Advance,
Mathias

**Comment by AlvinSchiller** [comment, marked as answer]
This has been added with https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/2266 in v2.6.0
